src/grid.cpp

```diff
@@ -15,7 +15,9 @@ Grid::Grid(const cv::Point2f &top_left_corner,
   if (m_CellSize <= 0) {
     std::ofstream debug_stream("debug_output.txt",
                                std::ios::app); // Debug output stream
-    debug_stream << "error: grid.cpp: line 18: m_Cell_Size is " << m_CellSize << "instead of positive number\n";
+    debug_stream << "[Grid::Grid(const cv::Point2f &, const cv::Point2f &, "
```

Comment:
You don't need to copy the entire function signature. The namespace and function name are enough.